nikola/plugins/task/tags.py

```diff
@@ -105,8 +105,7 @@ def gen_tasks(self):
             intersect = set(tags.keys()) & set(categories.keys())
             if len(intersect) > 0:
                 for slug in intersect:
-                    utils.LOGGER.error("Category '{0}' and tag '{1}' both have the same slug '{2}'!".format('/'.join(categories[slug]), tags[slug], slug))
```

Comment:
Because if a post was in more than one category it was shown as "cat1/cat2/cat3" and I think it's clearer to show it as "cat1, cat2, cat3"